src/include/sof/sof.h

```diff
@@ -53,6 +53,10 @@ struct sa;
 	__attribute__((unused))		\
 	typedef char assertion_failed_##MESSAGE[(COND) ? 1 : -1]
 
+/* concatenate preprocessing symbols with prior expansion */
+#define _CAT(x, y) x##y
+#define CAT(x, y) _CAT(x, y)
```

Comment:
I think this is already defined in meta programming

Reply:
See
https://github.com/thesofproject/sof/blob/33e4b993512751b2847e75e6b713e3a39c20afc0/src/include/sof/preproc.h#L60 for  `#define CAT(x, y) _CAT(x, y)`
https://github.com/thesofproject/sof/blob/33e4b993512751b2847e75e6b713e3a39c20afc0/src/include/sof/preproc-private.h#L91 for `#define _CAT(x, y) x##y`

Reply:
At the time of commiting these changes I wasn't aware that metaprogramming PR was already merged. I've corrected for that.